owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewAudioFragment.kt

```diff
@@ -285,11 +300,13 @@ class PreviewAudioFragment : FileFragment() {
 
             R.id.action_set_available_offline -> {
                 fileOperationsViewModel.performOperation(FileOperation.SetFilesAsAvailableOffline(listOf(file)))
+                Snackbar.make(requireView(), R.string.sync_file_nothing_to_do_msg, Snackbar.LENGTH_LONG).show()
```

Comment:
Uhm... not sure if showing the message `File contents already synchronized` when pressing on both `Set/Unset as available offline` options is suitable. This deserves another iteration, maybe with more specific messages. Same in the other preview classes

Reply:
I've used that string because it is used in file details screen. It can be used `The file was set as (un)available offline correctly` string instead. Also, it can be changed in `FileDetailsFragment` because it uses the same string.